washee_box/DockerfileOld

```diff
@@ -0,0 +1,54 @@
+# syntax=docker/dockerfile:1
+
+
```

Comment:
When the new dockerfile works, i think it will be ok to remove the old dockerfile